GliaWidgets/Interactor/Interactor.swift

```diff
@@ -113,29 +115,30 @@ extension Interactor {
     ) {
         state = .enqueueing
 
-        var options: CoreSdkClient.EngagementOptions? = nil
-
-        if mediaType == .audio || mediaType == .video {
-            options = .init(mediaDirection: .twoWay)
-        }
-
-        environment.coreSdk.queueForEngagement(
-            queueID,
-            visitorContext,
-            // shouldCloseAllQueues is `true` by default core sdk,
-            // here it is passed explicitly
-            true,
-            mediaType,
-            options
-        ) { [weak self] queueTicket, error in
-            if let error = error {
-                self?.state = .ended(.byError)
-                failure(error)
-            } else if let ticket = queueTicket {
-                if case .enqueueing = self?.state {
-                    self?.state = .enqueued(ticket)
+        let options = mediaType == .audio || mediaType == .video
```

Comment:
It would be great to add test ensuring that `enqueueForEngagement` calls `environment.coreSdk.configureWithConfiguration` and `environment.coreSdk.configureWithInteractor`. You can rely on failable mocks for this.

Reply:
fixed - [c531c39](https://github.com/salemove/ios-sdk-widgets/pull/230/commits/c531c39acba1d5f89a1064667c2ce13f10057488)